frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -168,6 +168,15 @@ export default function SettingsCluster() {
             : t('settings|Cluster Settings')
         }
         backLink
+        headerProps={{
+          actions: [
+            <Box justifyContent="flex-end" display={'flex'}>
```

Comment:
Is this box needed? I think the actions are already added to right, so maybe this can be skipped.